pkg/portal/graphql/sms_mutation.go

```diff
@@ -0,0 +1,104 @@
+package graphql
+
+import (
+	"encoding/json"
+
+	"github.com/authgear/authgear-server/pkg/api/apierrors"
+	"github.com/authgear/authgear-server/pkg/portal/model"
+	relay "github.com/authgear/graphql-go-relay"
+	"github.com/graphql-go/graphql"
+)
+
+var sendTestSMSInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "SendTestSMSInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"appID": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.ID),
+			Description: "App ID to test.",
+		},
+		"to": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "The recipient phone number.",
+		},
+		"providerConfiguration": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(smsProviderConfigurationInput),
+			Description: "The sms provider configuration.",
+		},
+	},
+})
+
+var smsProviderConfigurationInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "SMSProviderConfigurationInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"twilio": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.ID),
```

Comment:
The type is incorrect. Should it be SMSProviderConfigurationTwilioInput?

Reply:
Fixed in later commits